pkg/steps/clusterinstall/template.go

```diff
@@ -113,6 +115,33 @@ objects:
     - name: cluster-profile
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
+    - name: appci-release-bot-credentials
+      secret:
+        items:
+        - key: sa.release-bot.app.ci.config
+          path: sa.release-bot.app.ci.config
+        secretName: appci
+
+    initContainers:
+    # annotate the information about this job so that the release controller adds it to the upgrade graph
+    - name: annotate	
+      image: ${IMAGE_CLI}
+      env:
+      - name: RELEASE_IMAGE_INITIAL
+      - name: RELEASE_IMAGE_LATEST
```

Comment:
for what its worth, we also have chained update jobs like `release-openshift-origin-installer-e2e-aws-upgrade-4.1-to-4.2-to-4.3-to-4.4-nightly`, which will include several versions, not just two.  Not that those need to be added to the release controller's graph, but we will presumably want them managed with similar multi-step tooling at some point.